src/stripe.ios.ts

```diff
@@ -35,6 +35,38 @@ export class Stripe {
     );
   }
 
+  createSource(card: CardCommon, cb: (error: Error, source: Source) => void): void {
+    if (!card) {
+      if (typeof cb === 'function') {
+        cb(new Error('Invalid card'), null);
+      }
+      return;
+    }
+    
+    const sourceParams = STPSourceParams.cardParamsWithCard(card.native)
+
+    const apiClient = STPAPIClient.sharedClient();
+    apiClient.createSourceWithParamsCompletion(
+      sourceParams, callback(cb, (source) => <Source> {
```

Comment:
Use `source: STPSource` to find all the mapping problems.

Reply:
Sorry I do not understand this.